lib/java_buildpack/framework/contrast_security_agent.rb

```diff
@@ -49,7 +49,13 @@ def release
 
       # (see JavaBuildpack::Component::VersionedDependencyComponent#jar_name)
       def jar_name
-        "contrast-engine-#{@version.to_s.split('_')[0]}.jar"
+        boot_version = @version.to_s.split('_')[0]
```

Comment:
Depending on how you feel about this, the more idiomatic way to do this same thing would be to do

`boot_version = "#{@version[0]}.#{@version[1]}.#{@version[2]}"`

since the `_<QUALIFIER>` isn't a required part of the version spec.  The first three numeric components are required and guaranteed to exist.